dali/python/nvidia/dali/plugin/pytorch/__init__.py

```diff
@@ -652,65 +667,87 @@ def __next__(self) -> List[Dict[str, torch.Tensor]]:
                 else:
                     category_device[category] = torch_cpu_device
 
+            copy = not is_exec_dynamic
+
             pyt_tensors = dict()
-            for j, category in enumerate(self.output_map):
-                if (
-                    self._outputs_types is None
-                    or self._outputs_types[j] == DALIRaggedIterator.DENSE_TAG
-                ):
-                    pyt_tensors[category] = torch.empty(
-                        category_shapes[category],
-                        dtype=category_torch_type[category],
-                        device=category_device[category],
-                    )
-                else:
-                    pyt_tensors[category] = [
-                        torch.empty(
-                            shape,
+            if copy:
```

Comment:
OMG, GitHub made a big mess here. It's just indented and reformatted. Nothing more happened in the `if copy` branch.